Microsoft.Toolkit.Services/Services/LinkedIn/LinkedInDataProvider.cs

```diff
@@ -58,34 +54,16 @@ public class LinkedInDataProvider
         /// </summary>
         /// <param name="tokens">OAuth tokens for request.</param>
         /// <param name="requiredPermissions">Required permissions for the session.</param>
-        public LinkedInDataProvider(LinkedInOAuthTokens tokens, LinkedInPermissions requiredPermissions)
+        /// <param name="authentication">Authentication result interface.</param>
+        /// <param name="passwordManager">Password Manager interface, store the password.</param>
+        /// <param name="storageManager">Storage Manager interface.</param>
+        public LinkedInDataProvider(LinkedInOAuthTokens tokens, LinkedInPermissions requiredPermissions, IAuthenticationBroker authentication, IPasswordManager passwordManager, IStorageManager storageManager)
```

Comment:
This looks like a breaking change in a public api. There should be a UWP only constructor that will not create a breaking change.